pkg/monitor/monitor/monitor_manager.go

```diff
@@ -389,37 +396,15 @@ func (m *MonitorManager) removeIngressIfExist(monitor *v1alpha1.TidbMonitor, nam
 	return m.deps.TypedControl.Delete(monitor, ingress)
 }
 
-func (m *MonitorManager) patchTidbClusterStatus(tcRef *v1alpha1.TidbClusterRef, monitor *v1alpha1.TidbMonitor) error {
-	tc, err := m.deps.Clientset.PingcapV1alpha1().TidbClusters(tcRef.Namespace).Get(tcRef.Name, metav1.GetOptions{})
-	if err != nil {
-		return err
-	}
-	var mergePatch []byte
-	if tcRef != nil {
```

Comment:
The code here is to clear the TC status when the TidbMonitor is deleted, why remove this?

Reply:
I think this code is problematic, if tidbcluster can’t be found, we don’t need patch status.

Reply:
Yes, you're right, the code here is really a mess, if tcRef is nil, it will panic at the beginning of the function.